content/SCALE/SCALECLIReference/Auth/CLISessions.md

```diff
@@ -12,4 +12,225 @@ tags:
+
+{{< expand "Displaying if sessions are current" "v" >}}
+The `current` command has no required options. It returns a table showing which sessions are current. The rows of the table are relative to the rows returned from the `sessions` command.
+
+*Possible states:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true`   | Indicates the session is current (active). |
+| `false`  | Indicates the session is not current (inactive). |
+{{< /truetable >}}
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions current`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions current
++---------+
+| current |
++---------+
+| false   |
+| true    |
++---------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Internal Command
+
+The `internal` command displays whether sessions in the `sessions` list are internally created (via the web UI) or not (via SSH).
+
+{{< expand "Displaying if sessions are internal" "v" >}}
+The `internal` command has no required options. It returns a table showing if sessions are internal or not. The rows of the table are relative to the rows returned from the `sessions` command.
+
+*Possible states:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true`   | Indicates the session is internal (created via the web UI). |
+| `false`  | Indicates the session is not internal (created via SSH). |
+{{< /truetable >}}
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions internal`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions internal
++----------+
+| internal |
++----------+
+| false    |
+| true     |
++----------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Origin Command
+
+The `origin` command displays the login origin of the sessions in the `sessions` list.
+
+{{< expand "Displaying the login origin of sessions" "v" >}}
+The `origin` command has no required options. It returns a table showing the login origin of each session. The rows of the table are relative to the rows returned from the `sessions` command.
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions origin`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions origin
++---------------------------------------+
+| origin                                |
++---------------------------------------+
+| 10.234.15.210:15864                   |
+| UNIX socket (pid=2459768 uid=0 gid=0) |
++---------------------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Credentials Command
+
+The `credentials` command displays the credentials used to authenticate each session in the `sessions` list.
+
+{{< expand "Displaying how each session authenticated" "v" >}}
+The `credentials` command has no required options. It returns a table showing the credentials used to authenticate each session. The rows of the table are relative to the rows returned from the `sessions` command.
```

Comment:
same comment. Missing the enter command instruction. Also, tie in the table with the possible states so the user makes this connection better. For example, See the possible states table below for more information on what the credentials use to authenticate each session, or something similar.